src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java

```diff
@@ -668,6 +696,27 @@ public void matchingOnMultipartRequestBodyWithAContainsAndANegativeRegex() {
     assertThat(response.statusCode(), is(HTTP_OK));
   }
 
+  @Test
+  public void matchingOnMultipartRequestBodyWithNotContaining() {
+    stubFor(
+        post(urlEqualTo("/match/this/part/too"))
+            .withMultipartRequestBody(
+                aMultipart()
+                    .withHeader("Content-Type", containing("text/plain"))
```

Comment:
I presume you meant to test the `notContaining` matcher here?

Reply:
Good point. Thanks.